### ISSUE TITLE
Issue in Nesting a Draggable inside a Droppable

### ISSUE THREAD

**Issue by Karthik-B-06** [mohebifar/react-native-easy-dnd#6]
![Screenshot 2019-11-22 at 9 21 31 PM](https://user-images.githubusercontent.com/35562287/69475472-910d3f80-0df3-11ea-8b8b-54ccf01f2b62.png)

Any fix around it ?

**Comment by bmbknds** [comment]
@Karthik-B-06 are you trying to make a kanban app like trello ?

**Comment by lizhengnacl** [comment]
I had the same problem.

**Comment by bmbknds** [comment]
@lizhengnacl liz
I fixed by edit node module lib, on updateDraggable, instead of this.setState, i change to this.state.draggables = ... 
![Screen Shot 2019-12-27 at 4 28 04 PM](https://user-images.githubusercontent.com/29396046/71511808-34a3b100-28c6-11ea-9823-9093c7dd7420.png)

**Comment by Karthik-B-06** [comment]
@bmbknds Yeah something like trello kanban board. I am still on the search for proper implementation.

**Comment by squatto** [comment]
@Karthik-B-06 this isn’t RN but in case you haven’t come across it I wanted to share: https://github.com/atlassian/react-beautiful-dnd

**Comment by Karthik-B-06** [comment]
@squatto With react there are various options, searching for a solution in react-native.

**Comment by Nehal-29** [comment]
@Karthik-B-06 did you find anything ? 
Looking for the same.

**Comment by Karthik-B-06** [comment]
@Nehal-29 No, not yet. But I am constantly looking for react native apps that have Drag and Drop implemented in a section list.

**Comment by fkotsian** [comment]
If anyone's looking for this in 2021, this repo was pretty interesting for me: https://github.com/bear-junior/react-native-draganddrop-board/
